src/Tests/dotnet-new.Tests/CommonTemplatesTests.cs

```diff
@@ -165,8 +165,8 @@ public void NuGetConfigPermissions()
         public async void AotVariants(string name, string language)
         {
             // templates have not be updated to 9 yet "net8.0";
```

Comment:
This comment should be updated since this PR is updating templates to 9. We could remove the comment and hardcoded line or change the comment to say something like "template framework needs to be hardcoded here during the major version transition"